templates/amq-interconnect-1-basic.yaml

```diff
@@ -156,3 +198,27 @@ objects:
     name: configmap
   data:
     qdrouterd.conf: ${QDROUTERD_CONF}
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: ${APPLICATION_NAME}
+- kind: Role
+  apiVersion: v1
+  metadata:
+    name: ${APPLICATION_NAME}
+  rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["list"]
+- kind: RoleBinding
+  apiVersion: v1
+  metadata:
+    name: ${APPLICATION_NAME}
+  subjects:
+  - kind: ServiceAccount
+    name: ${APPLICATION_NAME}
+  roleRef:
+    kind: Role
+    name: ${APPLICATION_NAME}
+    namespace: ${APPLICATION_NAMESPACE}
```

Comment:
Is there a better way to find this?

Reply:
Not yet, support for namespace detection in templates is currently in the backlog: https://trello.com/c/zcRQFri0/916-3-namespace-and-username-template-parameters-templates

Apologies for merging this before checking in with you again. I thought I put this comment down already but it looks like I was only in "review" mode and never got published